uclaradio/uclaradio-blog
# Small API Refactoring

Changed the port 3010. Instead of recipes, we now have articles to checkout.

Testing
1. Follow "Setup Environment" according to README
2. Follow "Access Keystone" in README
3. Add articles to database in Keystone
4. Navigate to `http://localhost:3010/api/posts` to see all the articles
<img width="680" alt="screen shot 2018-12-22 at 1 32 00 am" src="https://user-images.githubusercontent.com/22732325/50372799-e4df2b00-0589-11e9-857f-e1b809451dd1.png">
5. In any of the objects, select an ID. Here, we select the id of the first object. 
<img width="800" alt="screen shot 2018-12-22 at 1 36 35 am" src="https://user-images.githubusercontent.com/22732325/50372812-28399980-058a-11e9-8927-5e02fc009347.png">

6. Navigate to `http://localhost:3010/api/posts/<ID>` and you should see the post for that ID
<img width="800" alt="screen shot 2018-12-22 at 1 38 58 am" src="https://user-images.githubusercontent.com/22732325/50372833-7cdd1480-058a-11e9-883e-4c7e2f944d4a.png">
